docs/reference/api.md

```
@@ -0,0 +1,101 @@
+    - Optional: `timeUnit`, `filters[]`, `groupBy[]`, `orderBy`, `limit`, `offset`
+  - Example:
+```bash
+curl -s -X POST https://api.databuddy.cc/v1/query/compile?website_id=WEBSITE_ID \
+  -H "content-type: application/json" \
+  -H "cookie: ...session..." \
+  -d '{
+    "projectId": "WEBSITE_ID",
+    "type": "summary_overview",
+    "from": "2025-01-01",
+    "to": "2025-01-31",
+    "timeUnit": "day",
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)